scripts/seed-database/write/flows.sql

```diff
@@ -51,6 +54,7 @@ SET
   copied_from = EXCLUDED.copied_from,
   analytics_link = NULL,
   status = EXCLUDED.status;
+  name = EXCLUDED.name;
```

Comment:
```suggestion
  status = EXCLUDED.status,
  name = EXCLUDED.name;
```

Reply:
A typo I've made myself many times when updating these scripts 😅 

You can actually test this locally by running `pnpm test-sync` which is a nice way of testing it.